### PR TITLE
Allow ioctl to show the list of actions for an account

### LINKED ISSUE #2567: Allow ioctl to show the list of actions for an account
**What would you like to be added**:
ioctl account actions ADDRESS|ALIAS [ SKIP [FIRST] ]

At least it returns (for each tx):
count + type + status + recipient + amount + hash
```
# lists all actions sorted from the most recent
$ ioctl account actions myAccount 

# lists all actions sorted from the most recent, skipping the latest 10
$ ioctl account actions myAccount 10 

# lists the most recent action
$ ioctl account actions myAccount 0 1 

# Example
12 transfer success io1jsljstd5xt62gzyde0usvqdjyf4vhh8peks9lf 12 IOTX ca01376611a11215589aaa664f71c8e466a5b5288438e50026f977e6928e2c48
```

**Why is this needed**:
When web explorers fail, we need a way to still be able to explore wallet transactions, then we use` ioctl account action hash `to get more details on each transaction

### PR DESCRIPTION
**Pull request by Liuhaai**
Fixed #2567

### CODE REVIEW

**Comment by dustinxie** [comment]
can you test this with couple of addresses to see if it works as expected?

**Comment by Liuhaai** [comment]
> can you test this with couple of addresses to see if it works as expected?



> can you test this with couple of addresses to see if it works as expected?

Yes, It works as expected.

File: ioctl/cmd/account/accountactions.go [hunk 1]
```diff
@@ -73,130 +76,51 @@ func accountActions(args []string) error {
...
-			ByAddr: &iotexapi.GetActionsByAddressRequest{
-				Address: addr,
-				Start:   numActions - *countLimit - skip,
-				Count:   *countLimit,
-			},
-		},
+		return output.NewError(output.ConvertError, "failed to pack in json", nil)
 	}
-	response, err := cli.GetActions(ctx, &requestGetAction)
+	resp, err := http.Post("https://iotexscout.io/apiproxy/api.ActionsService.GetAllActionsByAddress", "application/json",
```
Comment (millken) [inline ioctl/cmd/account/accountactions.go:87]: URL has moved to `https://iotex-analyser-api-mainnet.chainanalytics.org/api.ActionsService.GetActionsByAddress`

Comment (raullenchai) [reply ioctl/cmd/account/accountactions.go:87]: Better to make it an ioctl config.

File: ioctl/config/config.go [hunk 2]
```diff
@@ -87,6 +88,7 @@ func init() {
 	// Load or reset config file
 	var err error
 	ReadConfig, err = LoadConfig()
+	fmt.Printf("%+v\n", ReadConfig)
```
Comment (millken) [inline ioctl/config/config.go:91]: remove this

**Review by CoderZhi — APPROVED** [review]

**Pull request merged**